### PR TITLE
[Categories] Endpoint Detail Category OT245 - 42

### PR DESCRIPTION
**Pull request by R-Berrocal**
https://alkemy-labs.atlassian.net/browse/OT245-42

**Description**
AS admin user
I WANT to obtain information on an individual category
to be able to manage it

**Criteria of acceptance:**
GET to /categories/:id . It will show all the fields of a category based on the id sent by parameter. In the event that it does not exist, it will return an error with a 404 status code

**Evidence**
image
image
image
image

### CODE REVIEW

File: controllers/categories.js [hunk 1]
```diff
@@ -20,4 +20,21 @@ module.exports = {
       next(httpError)
     }
   }),
+
+  getId: catchAsync(async (req, res, next) => {
```
Comment (gabodmc) [inline controllers/categories.js]: getById

File: controllers/categories.js [hunk 2]
```diff
@@ -20,4 +20,21 @@ module.exports = {
...
+    try {
+      const category = await getCategoryId(req.params.id)
```
Comment (gabodmc) [inline controllers/categories.js]: getCategoryById

File: services/categories.js [hunk 3]
```diff
@@ -11,3 +11,16 @@ exports.getCategories = async () => {
     throw new ErrorObject(error.message, error.statusCode || 500)
   }
 }
+
+exports.getCategoryId = async (id) => {
```
Comment (gabodmc) [inline services/categories.js]: getCategoryById

**Review by gabodmc — CHANGES_REQUESTED** [review]
Lo veo bien. Solo unos cambios

**Review by gabodmc — APPROVED** [review]
Buenas correcciones. Adentrol el PR!

**Review by itsnicolearias — APPROVED** [review]

**Review by miguelpuente — APPROVED** [review]

**Pull request merged**